hybrasyl/Objects/Monster.cs

```diff
@@ -261,6 +261,7 @@ public void Cast(Creature target)
             var castable = World.WorldData.Get<Castable>(creatureCastable.Value);
             if (target is Merchant) return;
             UseCastable(castable, target);
+            Condition.Casting = false;
```

Comment:
This should be handled in the UseCastable method.

Reply:
Good point. Fixed.